src/System.Security.Cryptography.Encoding/tests/DerEncoderTests.cs

```diff
@@ -398,5 +398,71 @@ public static void CheckUtcTimeEncoding_Local_Static()
 
             Assert.Equal(encodedExpected, encodedLocal);
         }
+
+        [Fact]
+        public static void TestConstructSegmentedSequence()
+        {
+            byte[][] expected =
+            {
+                /* Tag */     new byte[] { 0x30 },
+                /* Length */   new byte[] { 0x07 },
+                /* Load */ new byte[] { 0x02, 0x01, 0x00, 0x02, 0x02, 0x01, 0x00},
+            };
+
+            byte[][] encoded = DerEncoder.ConstructSegmentedSequence(
+                DerEncoder.SegmentedEncodeUnsignedInteger(new byte[] { 0x00 }),
+                DerEncoder.SegmentedEncodeUnsignedInteger(new byte[] { 0x01, 0x00 }));
+
+            Assert.Equal(expected.Length, encoded.Length);
+
+            for (int i = 0; i < encoded.Length; i++)
+            {
+                Assert.Equal<byte>(expected[i], encoded[i]);
+            }
+        }
+
+        [Fact]
+        public static void TestConstructSegmentedImplicitSequence()
+        {
+            byte[][] expected =
+            {
+                /* Tag */     new byte[] { 0xA1 },
+                /* Length */   new byte[] { 0x07 },
+                /* Load */ new byte[] { 0x02, 0x01, 0x00, 0x02, 0x02, 0x01, 0x00},
+            };
+
+            byte[][] encoded = DerEncoder.ConstructSegmentedImplicitSequence(1,
+                DerEncoder.SegmentedEncodeUnsignedInteger(new byte[] { 0x00 }),
+                DerEncoder.SegmentedEncodeUnsignedInteger(new byte[] { 0x01, 0x00 }));
+
+            Assert.Equal(expected.Length, encoded.Length);
+
+            for (int i = 0; i < encoded.Length; i++)
+            {
+                Assert.Equal<byte>(expected[i], encoded[i]);
+            }
+        }
+
+        [Fact]
+        public static void TestConstructSegmentedExplicitSequence()
+        {
+            byte[][] expected =
+            {
+                /* Tag */     new byte[] { 0xA2 },
+                /* Length */   new byte[] { 0x09 },
+                /* Load (Has the sequence inside) */ new byte[] { 0x30, 0x07, 0x02, 0x01, 0x00, 0x02, 0x02, 0x01, 0x00},
+            };
+
+            byte[][] encoded = DerEncoder.ConstructSegmentedExplicitSequence(2,
```

Comment:
I completely missed that there was a 2, at the end of this line (and a 1, above).  Move the argument down to the next line, please.